src/pages/login/index.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import Image from 'next/image';
+import styles from "./login.module.css";
+import CustomizableAside from "../../components/customizableAside";
+
+
+const { useState } = React;
+
+function LoginForm() {
+
+    const [values, setValues] = useState({
+        email: "",
+        password: "",
+    });
+
+    const [error, setError] = useState(undefined);
+    const handleSubmit = async (e) => {
+        e.preventDefault();
+
+        try {
+            const response = await axios.post("/login", values);
+            console.log(response.data);
+            // handle success.
+        } catch (error) {
+            console.error(error);
+            // handle error (example).
+            setError("An error occurred, try again...");
+        }
+        
+
+    }
+    
+    
+    return (
+        <div className={styles.loginPage}>
+
+            <div className={styles.asideContainer}>
+            <CustomizableAside
+                    src={"/images/mainAvatar.png"}
+                    width={110}
+                    height={110}
+                    alt={"avatar"}
+                    asideCustomClass={styles.asideContain}
+                    headerText={"Lorem ipsum lorem ipsum"}
```

Comment:
I think this is ok for now, but I would put all of these values as variables at the top